azure-pipelines-deploy.yml

```diff
@@ -60,6 +60,12 @@ resources:
 extends:
   template: stages/wrapper_cd.yml@templates
   parameters:
+    automaticDeployments:
```

Comment:
How does this work alongside the pipeline trigger? At the moment on main this pipeline is triggered after build.

```
  pipelines:
    - pipeline: build
      source: Appeals Service Build
      trigger:
        branches:
          include:
            - main
 ```

Reply:
So my understanding is:

This object gets passed through to wrapper_cd in common as a param and overwrites the default there which only does dev:
https://github.com/Planning-Inspectorate/common-pipeline-templates/blob/main/stages/wrapper_cd.yml

Then it loops over the auto deployment envs to deploy on CI builds